advanced_functionality/autogluon-tabular-containers/AutoGluon_Tabular_SageMaker_Containers.ipynb

```diff
@@ -0,0 +1,735 @@
+{
```

Comment:
<p>If you are going to mention the specific 0.3.1 image version, do you want to specify that same version in the AutoGluonTraining()? Otherwise it will use a later version when released in the future and the text will become out of date.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2957/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Updated images handling classes to support `framework_version` arg; removed specific version from the text.